sklearn/model_selection/_search.py

```diff
@@ -882,8 +889,11 @@ class GridSearchCV(BaseSearchCV):
         FitFailedWarning is raised. This parameter does not affect the refit
         step, which will always raise the error.
 
-    return_train_score : boolean, default=True
-        If ``'False'``, the ``cv_results_`` attribute will not include training
+    return_train_score : boolean, optional
+        Current default is ``'warn'``, which behaves as ``True`` in addition
+        to raising a warning. That default will be changed to ``False``
+        in 0.21.
+        If ``False``, the ``cv_results_`` attribute will not include training
```

Comment:
This docstring does not match the other two. You don't have the "Computing training scores is used to get insights ..." section.

Reply:
Added.